.github/workflows/ci-test-integration.yml

```
@@ -12,6 +12,7 @@ on:
 
 jobs:
   build:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)